boxoffice/models/category.py

```diff
@@ -1,49 +1,23 @@
-from decimal import Decimal
 from boxoffice.models import db, BaseScopedNameMixin
-from boxoffice.models.event import Event
+from boxoffice.models import ItemCollection
 
-__all__ = ['Category', 'CATEGORY_STATUS', 'CATEGORY_STATUS_CODES']
+__all__ = ['Category']
 
-class CATEGORY_STATUS:
-	DRAFT = 1
-	LIVE = 2
-	EXPIRED = 3
-
-CATEGORY_STATUS_CODES = [
-    [1, "Draft"],
-    [2, "Live"],
-    [3, "Expired"]
-]
 
 class Category(BaseScopedNameMixin, db.Model):
-    """
-    Categories are classes of items that can be purchased, along with inventory available, unit price and tax rate
-    Categories can be in Draft, Live or Expired state.
-    """
     __tablename__ = 'category'
+    __uuid_primary_key__ = True
```

Comment:
Why are we using UUID primary keys on low volume tables like categories? UUIDs only make sense for high volume tables with inserts happening on multiple servers.